### PR TITLE
ESB 5.0.0 p2 updates

### PR DESCRIPTION
**Pull request by jagathsisira**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -570,12 +610,15 @@
                                     org.wso2.carbon.analytics-common:org.wso2.carbon.event.processor.manager.commons.feature:${carbon.analytics-common.version.5.0.10}
                                 </featureArtifactDef>
                                 <featureArtifactDef>
-                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.receiver.feature:${carbon.analytics-common.version.5.0.10}
```
Comment (callkalpa) [inline pom.xml:755]: Why have we removed this feature?

Comment (jagathsisira) [reply pom.xml:755]: It was not removed, but moved to line 619 since it was not in alphabetical order.

File: pom.xml [hunk 2]
```diff
@@ -749,22 +787,25 @@
                                     org.wso2.carbon.analytics-common:org.wso2.carbon.event.processor.manager.commons.feature:${carbon.analytics-common.version.5.1.0}
                                 </featureArtifactDef>
                                 <featureArtifactDef>
-                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.receiver.feature:${carbon.analytics-common.version.5.0.3}
+                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.publisher.aggregate.feature:${carbon.analytics-common.version.5.1.1}
```
Comment (callkalpa) [inline pom.xml]: This should come before org.wso2.carbon.event.processor.manager.*

File: pom.xml [hunk 3]
```diff
@@ -749,22 +787,25 @@
...
                                 </featureArtifactDef>
                                 <featureArtifactDef>
-                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.receiver.feature:${carbon.analytics-common.version.5.0.10}
+                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.publisher.feature:${carbon.analytics-common.version.5.0.3}
```
Comment (callkalpa) [inline pom.xml]: This should come before org.wso2.carbon.event.processor.manager.*

File: pom.xml [hunk 4]
```diff
@@ -749,22 +787,25 @@
...
                                 </featureArtifactDef>
                                 <featureArtifactDef>
-                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.receiver.feature:${carbon.analytics-common.version.5.1.0}
+                                    org.wso2.carbon.analytics-common:org.wso2.carbon.event.publisher.feature:${carbon.analytics-common.version.5.0.10}
```
Comment (callkalpa) [inline pom.xml]: This should come before org.wso2.carbon.event.processor.manager.*

**Pull request merged**